bfdd/bfd.c

```diff
@@ -1178,8 +1178,13 @@ const char *bs_to_string(const struct bfd_session *bs)
 		pos += snprintf(buf + pos, sizeof(buf) - pos, " vrf:%s",
 				bs->key.vrfname);
 	if (bs->key.ifname[0])
-		pos += snprintf(buf + pos, sizeof(buf) - pos, " ifname:%s",
-				bs->key.ifname);
+		snprintf(buf + pos, sizeof(buf) - pos, " ifname:%s",
+			 bs->key.ifname);
```

Comment:
I think a cleaner solution to silence the SA warning would be to add a  `(void)pos;` line after this one, like was done in commit e3df3ba6d4ea. Anyone agrees/disagrees?

Reply:
Sure, makes sense to me.

Reply:
fixed